src/webgl/context-state.js

```diff
@@ -317,25 +318,25 @@ const GL_STATE = {
   [GL.UNPACK_ALIGNMENT]: {
     type: GLint,
     params: GL.UNPACK_ALIGNMENT,
-    setter: (gl, value) => gl.pixelStorei(value)
+    setter: (gl, value) => gl.pixelStorei(GL.UNPACK_ALIGNMENT, value)
```

Comment:
Good fix! 

Gave me an idea, maybe the setter could get the flag as an argument, I'll look into that.